modules/custom/az_person/config/install/core.entity_view_display.node.az_person.default.yml

```diff
@@ -19,11 +19,192 @@ dependencies:
     - field.field.node.az_person.field_az_titles
     - node.type.az_person
   module:
+    - field_group
     - file
     - link
+    - smart_title
     - telephone
     - text
     - user
+third_party_settings:
+  smart_title:
+    enabled: true
+    settings:
+      smart_title__link: false
+      smart_title__tag: h1
+      smart_title__classes:
+        - node__title
```

Comment:
Do you know if the `node__title` class is necessary? I tried deleting it in Chrome and didn't notice any difference.

Reply:
Based on 2022-12-02 meeting discussion, we're thinking it would be best to leave this class for now in case anyone is targeting it for any reason, etc.

Reply:
We decided to leave the `node__title` class for now.